### ISSUE TITLE
CUDA Error: no kernel image is available for execution on the device

### ISSUE THREAD

**Issue by sharmin-sultana-sheuly** [AlexeyAB/darknet#8154]
Dear @AlexeyAB

I downloaded Darknet from the below site
https://github.com/AlexeyAB/darknet/archive/master.zip
My system configurations are:

Windows 10
GPU:NVIDIA Quadro M1200
OpenCV version: 4.5.3
CUDA-version: 11040
However it is showing error : CUDA Error: no kernel image is available for execution on the device (please see the attached document for the whole output).
[error.docx](https://github.com/AlexeyAB/darknet/files/7324976/error.docx)

Previously it showed errors : opencv_core453.dll,opencv_highgui453.dll,opencv_imgcodecs453.dll,opencv_imgproc453.dll,opencv_videoio453.dll,pthreadVC2.dll are missing.
I solved the problems by copying these libraries in the darknet-master folder.

**However, I could not solve the error ,CUDA Error: no kernel image is available for execution on the device**
Additionally I searched in the already solved issues. However, no clues.

Please Hepl me.

Regards
Sharmin

**Comment by stephanecharette** [comment]
Isn't that version quite old?  Is there a reason why you didn't take the latest from master?

**Comment by stephanecharette** [comment]
> Isn't that version quite old? Is there a reason why you didn't take the latest from master?

Nevermind, I see that does grab the latest from master.

**Comment by sharmin-sultana-sheuly** [comment]
Hi @cenit 

based on your suggestion I built darknet using build.ps1 script.
However when I run the command
darknet.exe detect cfg/yolov3.cfg yolov3.weights data/dog.jpg
it is still showing error, **CUDA Error: no kernel image is available for execution on the device**
I attached the attached the command prompt output.

Regards
Sharmin
[error2.docx](https://github.com/AlexeyAB/darknet/files/7340724/error2.docx)

**Comment by sharmin-sultana-sheuly** [comment]
Hi @AlexeyAB 

Thanks a lot.

For changing CC (CUDA Compute Capability) 5.0 in Cmake
I performed the below procedure:
a. Located CMakeLists.txt file in C:\Program Files\CMake\share\cmake-3.21\Modules\FortranCInterface
b. Added the lines : set_property(TARGET myTarget PROPERTY CUDA_ARCHITECTURES 50)
target_link_libraries(FortranCInterface PUBLIC myTarget)

However, still the CMAKE_CUDA_ARCHITECTURES: 52 (See the attached error3.docx)
I don’t know which file should contain these commands, also what should I do afterwards to make it work?
I never worked with CMake. Additionally there is no help from CMake forum.
Stuck in this simple step for hours :( Was embarrassed to ask you again. 

[error3.docx](https://github.com/AlexeyAB/darknet/files/7354372/error3.docx)

Regards
Sharmin
